src/test/java/com/hotels/mutantswarm/exec/OutcomeTest.java

```diff
@@ -54,13 +63,60 @@ public void testStateKilled(){
   
   @Test
   public void testStateSurvived(){
-    List<TestOutcome> testOutcomes = asList(testOutcome1, testOutcome2, testOutcome3);
     when(testOutcome1.getState()).thenReturn(MutantState.SURVIVED);
     when(testOutcome2.getState()).thenReturn(MutantState.SURVIVED);
     when(testOutcome3.getState()).thenReturn(MutantState.SURVIVED);
     
     Outcome outcome = new Outcome(mutant, mutation, testOutcomes);
     assertThat(outcome.getState(), is(MutantState.SURVIVED));
   }
+  
+  @Test
+  public void checkToString(){
+    Outcome outcome = new Outcome(mutant, mutation, testOutcomes);
+    String result = outcome.toString();
+    assertThat(result,is("Outcome [mutant=mutant, mutation=mutation, state=SURVIVED, testOutcomes=[testOutcome1, testOutcome2, testOutcome3]]"));
+  }
+  
+  @Test
+  public void equalsNull() {
+    Outcome outcome = new Outcome(mutant, mutation, testOutcomes);
+    assertThat(outcome.equals(null), is(false));
+  }
+
+  @Test
+  public void equalsMutantNull() {
+    Outcome outcome = new Outcome(null, mutation, testOutcomes);
+    Outcome outcome2 = new Outcome(mutant, mutation, testOutcomes);
+    assertThat(outcome.equals(outcome2), is(false));
+  }
+  
+  @Test
+  public void equalsMutantionNull() {
+    Outcome outcome = new Outcome(mutant, null, testOutcomes);
+    Outcome outcome2 = new Outcome(mutant, mutation, testOutcomes);
+    assertThat(outcome.equals(outcome2), is(false));
+  }
+  
+  @Test
+  public void equalsSame() {
+    Outcome outcome = new Outcome(mutant, mutation, testOutcomes);
+    Outcome outcome2 = new Outcome(mutant, mutation, testOutcomes);
+    assertThat(outcome.equals(outcome2), is(true));
```

Comment:
This is a good place to add a symmetric test: 

`assertThat(outcome2.equals(outcome), is(true));`

and a test on the hash:

`assertThat(outcome.hashCode() == outcome2.hashCode(), is(true));`